### DISCUSSION TITLE
Character Turn Around

### DISCUSSION THREAD

**Discussion by DreamBagAttic** [keiwando/toonsquid#654] [Q&A]
I’m new to animation. Trying to learn how to do a character turn around in toon squid but can’t find any tutorials / videos. Any help is greatly appreciated. Thanks

**Comment by Cat-Butler** [comment]
For a turn around for a character sheet, there isn't really anything special to ToonSquid, it would be much the same as in any paint app.
One possibly useful tip would be to set aside a layer for "consistency lines". 
Examples, see the lines and notes for: 
- [Donald Duck](https://www.sequentialartistsworkshop.org/blog/2020/9/21/keep-characters-consistent-or-not)
- [Meg (Disney's Hercules)](https://forum.svslearn.com/topic/1329/a-course-on-keeping-characters-consistent/5?lang=en-US)

Also useful without the consistency lines, as a reference if you ever need to animate a character from that angle, examples:
- [Widget](https://www.flickr.com/photos/84568447@N00/2978931790), [Walden](https://www.flickr.com/photos/84568447@N00/2978074757/in/album-72157608423433061), and [Wubbzy](https://www.flickr.com/photos/84568447@N00/2978931078/in/album-72157608423433061)

You could do it frame-by-frame, where again a layer with consistency lines (and perhaps onion skins) might be helpful.

As an animation exercise, I'm not sure that it is worth it. As far as I can tell, the animated turn around was introduced by 3D animators to show off their shiny new models. The rare times I've seen it in 2D they've usually been [lampshading](https://en.wiktionary.org/wiki/lampshade) the process as a mild mockery of the 3D cliché.

If you want a character spin as part of their movement, [at an entertaining speed](https://www.youtube.com/watch?v=ynBENQJBQbE&t=477s) there are [very few frames](https://www.reddit.com/r/animation/comments/1cq0rj5/how_smear_frames_are_used_and_how_they_work_for/) actually involved, and some intermediate frames are probably going to be a [blur](https://www.traditionalanimation.com/2017/smear-speed-motion-blur-effects-in-animation/), [smear](https://idearocketanimation.com/8857-animation-techniques-smear/), or [multiples](https://www.flickr.com/photos/84568447@N00/2978074659/in/album-72157608423433061) (with multiple eyes/limbs/etc on the same frame).

**Reply by DreamBagAttic** [reply]
Thank you for your time in responding here. I'll try this out

**Reply by diegostoliar** [reply]
I guess people opinions and studio workflows may vary. On the projects I worked on we always do turn arounds. 

If it's a cutout show the turn usually has a minimum of 4 poses (front, quarter front, side, Quarter Back) to 8 poses (Almost front, Almost side, Back, Almost back)... Those extra ones depend on the project complexity or character complexity.

If it's a full animation and the budget is tight you can do less, for example just do a full turn for the head, but less drawings for the body... again this also depends on the complexity of the design and what you need to explain with drawings to the team.

It does help to use an animation program to work on turns, since you can flip the drawings and use onion skin to keep proportions.

I neat trick is to create some guides to help you in another layer... if the app does not have guides you can just draw lines. I also think its a good idea to draw the main volume of the shapes in the guide window.

Unlike 3D we never do the turn on a T or A pose, for 2D shows we use a rest pose, where the character is just idle.

Keeping in mind also that turn is just 1 item on character sheets, "dos and don't" can be as or more important than the turn.

**Reply by DreamBagAttic** [reply]
Awesome. Thank you @diegostoliar. What software do you most commonly use for 2D?

**Reply by diegostoliar** [reply]
I  mostly use Toonboom Harmony at work, at home, more and more I use Toonsquid (seriously, I really like it)... also learning a bit of Clip Studio.
